Clinical-Genomics/scout
# Hide old archived observations panel with config parameter

**####### PLEASE WAIT WITH REVIEW. THIS IS STILL A WORK IN PROGRESS. I'm PLANNING TO ADD SOME CODE TO PARSE THE DATE RELATIVE TO THE LOQUSDB ARCHIVE USED TO CHECK LOCAL FREQUENCIES ##########**

Introduces a config parameter `SHOW_ARCHIVED_OBSERVATIONS` that when is set to False will hide the archived local observations panel on the variant page. Partial fix to #2681

If `SHOW_ARCHIVED_OBSERVATIONS` is missing from config file, or if it's set to True, the. then archived observations will be shown 

**How to test**:
1. Locally, in config file:
- Comment the line containing SHOW_ARCHIVED_OBSERVATIONS param
- Set the param to True
- Set the param to False

Check the difference on the variant page

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
